src/slides/1-demos/documentation/styles.css

```
@@ -15,6 +15,10 @@ body {
 	margin: auto;
 }
 
+p {
+	line-height: 20px;
```

jamais en px les line-height (comme les font-size), il vaut mieux mettre une valeur sans unité qui suit la font-size
```suggestion
	line-height: 1.25;
```